README.md

```diff
@@ -55,11 +55,11 @@ __Tags__: Tags are key-value pairs that are specified in the configuration file(
 ## Rest API to get the RCAs
 * To get response for all the available RCA, use:
 ```
-curl --url "localhost:9650/_opendistro/_performanceanalyzer/rca" -XGET
+curl --url "localhost:9650/_plugins/_performanceanalyzer/rca" -XGET
```

Comment:
Can we update this to port 9600. Thanks!

Reply:
Done.

Reply:
Done.